referenceframe/frame_system.go

```diff
@@ -244,10 +244,13 @@ func (sfs *simpleFrameSystem) AddFrame(frame, parent Frame) error {
 	return nil
 }
 
+// NOTE: What is a transforable object?
```

Comment:
Conceptually, thing A is transformable if it is spatially located somewhere relative to thing B; thus, if a framesystem knows where thing be is relative to thing C, then thing A can be transformed such that it is somewhere relative to thing C.